agent/src/main/java/kanela/agent/Kanela.java

```diff
@@ -16,44 +16,89 @@
 
 package kanela.agent;
 
+import io.vavr.collection.List;
+import kanela.agent.api.instrumentation.listener.InstrumentationRegistryListener;
+import kanela.agent.api.instrumentation.replacer.ClassReplacer;
+import kanela.agent.builder.KanelaFileTransformer;
+import kanela.agent.circuitbreaker.SystemThroughputCircuitBreaker;
+import kanela.agent.reinstrument.Reinstrumenter;
+import kanela.agent.util.BootstrapInjector;
+import kanela.agent.util.banner.KanelaBanner;
+import kanela.agent.util.classloader.InstrumentationClassPath;
+import kanela.agent.util.conf.KanelaConfiguration;
+import kanela.agent.util.jvm.OldGarbageCollectorListener;
+import lombok.val;
+
 import java.lang.instrument.Instrumentation;
 
+import static kanela.agent.util.Execution.runWithTimeSpent;
+
+public class Kanela {
```

Comment:
`final` should not have been removed, and `@Value` was removed because this class is not a real candidate for a value class, it's only 2 pieces of state are static and mutable, it just didn't seem to make sense. `final` is back!